### PR TITLE
0.13.0 changelog

### PR DESCRIPTION
**Pull request by OpenSauce04**
### Summary:

Updated the changelog to include all notable changes from the 0.13.0 update

### CODE REVIEW

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,17 @@
 # Changelog
-
+### v0.13.0
+- Added preset support! Create and share your own custom OSBES adjustments.
```
Comment (jebbyk) [inline docs/CHANGELOG.md:3]: it was not in 0.13.0
it only appeared in 0.13.1

**Review by jebbyk — COMMENTED** [review]
You may also add changes for 0.13.1

**Comment by OpenSauce04** [comment]
What else was even added in 0.13.1?

**Comment by jebbyk** [comment]
quality settings was implemented in 0.13.1 too. 0.13.0 is only refactoring, improving clouds and reflections

**Comment by OpenSauce04** [comment]
I know, so that's why I put it under 0.13.1
I don't see the problem here :P
Did you even look at the file?

**Comment by jebbyk** [comment]
oh sorry :-)

**Pull request merged**